client/components/videos-ui/index.jsx

```diff
@@ -118,26 +121,39 @@ const VideosUi = ( {
 				) }
 				<div className="videos-ui__header-content">
 					<div className="videos-ui__titles">
-						<h2>{ translate( 'Watch five videos.' ) }</h2>
-						<h2>{ translate( 'Save yourself hours.' ) }</h2>
+						<h2>{ title ? title : translate( 'Watch five videos.' ) }</h2>
+						<h2>{ subtitle ? subtitle : translate( 'Save yourself hours.' ) }</h2>
 					</div>
 					<div className="videos-ui__summary">
 						<ul>
-							<li>
-								<Gridicon icon="checkmark" size={ 18 } />{ ' ' }
-								{ translate( 'Learn the basics of blogging' ) }
-							</li>
-							<li>
-								<Gridicon icon="checkmark" size={ 18 } />{ ' ' }
-								{ translate( 'Familiarize yourself with WordPress' ) }
-							</li>
-							<li>
-								<Gridicon icon="checkmark" size={ 18 } /> { translate( 'Upskill and save hours' ) }
-							</li>
-							<li>
-								<Gridicon icon="checkmark" size={ 18 } />{ ' ' }
-								{ translate( 'Set yourself up for blogging success' ) }
-							</li>
+							{ ( ! options || options[ 0 ] ) && (
```

Comment:
What if we iterate over the options array? This way, we can simplify the logic. The existing hardcoded strings can be passed as regular parameters where it's used. Or... we could check if the options is empty and populate the variable with a default array.

Personally I would go with the first approach but if there a lot of usages of this component (I doubt), option 2 is also fine. :)

Reply:
Yes, I thought about that but first i couldn't use translate() function with variable. But now first I'm creating array with translated string and now I can iterate.